app/views/shared/_spotlight_search.html.erb

```diff
@@ -0,0 +1,27 @@
+<% main_resource = request.path.split("/").last.presence_in(%w[talks speakers events]) || "talks" %>
```

Comment:
depending on the location the `enter` key will navigate to different resource

When on the speaker page, type `da` + enter sends you to `/speaker?s=da`

open to discuss this UX if not clear